harvest/batch-import-marc.sh

```diff
@@ -70,6 +77,13 @@ then
   exit 1
 fi
 
+# Check MAX_BATCH_COUNT is a positive integer
+if ! [[ "$MAX_BATCH_COUNT" =~ ^[1-9][0-9]*$ ]]
```

Comment:
@damien-git, I adjusted this regex since it was still possible to say `-x 0`, which led to an xargs error instead of a more meaningful message. Obviously this will prevent someone from doing something like `-x 0110` which might technically be legal but doesn't really make sense, so I'm not too worried about it.

Reply:
OK